test/metabase/query_processor_test/nested_queries_test.clj

```diff
@@ -235,10 +235,10 @@
 ;; e.g. the ORDER BY in the source-query should refer the 'stddev' aggregation, NOT the 'avg' aggregation
 (expect
   {:query (str "SELECT avg(\"stddev\") AS \"avg\" FROM ("
-                   "SELECT STDDEV(\"PUBLIC\".\"VENUES\".\"ID\") AS \"stddev\", \"PUBLIC\".\"VENUES\".\"PRICE\" AS \"PRICE\" "
-                   "FROM \"PUBLIC\".\"VENUES\" "
-                   "GROUP BY \"PUBLIC\".\"VENUES\".\"PRICE\" "
-                   "ORDER BY \"stddev\" DESC, \"PUBLIC\".\"VENUES\".\"PRICE\" ASC"
+                   "SELECT STDDEV(\"t1\".\"ID\") AS \"stddev\", \"t1\".\"PRICE\" AS \"PRICE\" "
+                   "FROM \"PUBLIC\".\"VENUES\" \"t1\" "
+                   "GROUP BY \"t1\".\"PRICE\" "
+                   "ORDER BY \"stddev\" DESC, \"t1\".\"PRICE\" ASC"
```

Comment:
I'm not sure how I feel about changing a test shared across all of our DBs just to satisfy one new driver. I'd be surprised if this didn't end up breaking tests for other drivers

Reply:
It's not just one driver though, this change affects all drivers. We should make sure it works with all of them, so I think changing the test makes sense. Please do help with testing other drivers (I've tested H2, Spark SQL, Drill, Postgres).

Reply:
Ended up breaking BigQuery 😭

Reply:
fixed in #7323